mne/annotations.py

```diff
@@ -900,12 +900,32 @@ def _combine_annotations(one, two, one_n_samples, one_first_samp,
 def _handle_meas_date(meas_date):
     """Convert meas_date to datetime or None.
 
+    See :func: '_handle_mas_date_nonUTC'
+    Note: Prefer '_handle_mas_date_nonUTC', but leaving this here to make
+    sure nothing gets messed up by adding return values.
+    """
+    meas_date, _, _ = _handle_meas_date_nonUTC(meas_date)
+    return meas_date
```

Comment:
I don't see the point of having this very thin wrapper around _handle_meas_date_nonUTC.  I think it's better to just change how the old `_handle_meas_date` function worked (including returning extra values) and then do a `git grep _handle_meas_date` to find all the places we use it, and update them to either ignore the extra return values or make use of them (if appropriate).